### PR TITLE
Add type ServerInfo, add SInfo entry to ConnConfig

### PR DESCRIPTION
**Pull request by koba-e964**
This PR allows users of go-rtmp to change the behavior of rtmp server. Namely, they can change the server's name and its version in its rtmp response.

### CODE REVIEW

**Comment by yutopp** [comment]
Thank you for your PR!

I have exported `ServerConnectResultProperties` and `ServerConnectResultData` as public global variables for these usage. Thus, you can overwrite these variables directly by yourself.

An assumed usage is like this,
```
// In your package file which uses go-rtmp.
func init() {
    rtmp.ServerConnectResultProperties.FMSVer = "hogehoge"
}
```

Isn't it suitable for your use?

**Comment by koba-e964** [comment]
Well, I didn't notice that these variables (`ServerConnectResultProperties` and `ServerConnectResultData`) were exported. (Just because I wasn't well accustomed to Go.)
Ah, yes, the code snippet you gave is what I intended to do, and is completely suitable.

By the way, I prefer adding fields in the configuration struct (`ConnConfig`) to letting users overwrite global variables. That's because:
- if two or more users overwrite global variables, their changes may conflict,
- and therefore I believe that passing parameters as function arguments, not as global variables, is a better option.

What would you say about this?

File: conn.go [hunk 1]
```diff
@@ -49,6 +49,7 @@ type ConnConfig struct {
 	ControlState StreamControlStateConfig
 
 	Logger logrus.FieldLogger
+	SInfo  ServerInfo
```
[the remark this answers is not in the document]
Comment (koba-e964) [reply conn.go]: `ResponsePreset` sounds good to me. I will rename.

[the remark this answers is not in the document]
Comment (koba-e964) [reply conn.go]: Done https://github.com/yutopp/go-rtmp/pull/28/commits/7bc4ffae79a012a7cbc534fd96c2803da233aa9f

File: server_info.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	Mode:         1,                 // TODO: fix
+}
+
+// ServerConnectResultData Sent to clients as result when Connect message is received
+var serverConnectResultData = map[string]interface{}{
+	"type":    "go-rtmp",
+	"version": "master", // TODO: fix
+}
+
+// ServerInfo is an interface to provider server info.
```
[the remark this answers is not in the document]
Comment (koba-e964) [reply server_info.go]: Done https://github.com/yutopp/go-rtmp/pull/28/commits/7bc4ffae79a012a7cbc534fd96c2803da233aa9f

**Pull request merged**